packages/fiori/src/NotificationListGroupItem.ts

```diff
@@ -0,0 +1,246 @@
+ * <li><code>Priority</code> icon to display the priority of the group</li>
+ * <li><code>TitleText</code> to entitle the group</li>
+ * <li>Custom actions - with the use of <code>ui5-notification-action</code></li>
+ * <li>Items of the group</li>
+ * </ul>
+ *
+ * <h3>Usage</h3>
+ * The component can be used in a standard <code>ui5-list</code>.
+ *
+ * <h3>CSS Shadow Parts</h3>
+ *
+ * <ui5-link target="_blank" href="https://developer.mozilla.org/en-US/docs/Web/CSS/::part">CSS Shadow Parts</ui5-link> allow developers to style elements inside the Shadow DOM.
+ * <br>
+ * The <code>ui5-li-notification-group</code> exposes the following CSS Shadow Parts:
+ * <ul>
+ * <li>title-text - Used to style the titleText of the notification list group item</li>
+ * </ul>
+ *
+ * <h3>ES6 Module Import</h3>
+ *
+ * <code>import "@ui5/webcomponents/dist/NotificationListGroupItem.js";</code>
+ * <br>
+ * <code>import "@ui5/webcomponents/dist/NotificationAction.js";</code> (optional)
+ * @constructor
+ * @author SAP SE
+ * @alias sap.ui.webc.fiori.NotificationListGroupItem
+ * @extends sap.ui.webc.fiori.NotificationListItemBase
+ * @tagname ui5-li-notification-group
+ * @since 1.0.0-rc.8
+ * @appenddocs sap.ui.webc.fiori.NotificationAction
+ * @implements sap.ui.webc.main.IListItem
+ * @public
+ */
+@customElement("ui5-li-notification-group")
+@languageAware
+
+/**
+ * Fired when the <code>ui5-li-notification-group</code> is expanded/collapsed by user interaction.
+ *
+ * @event
+ * @public
+ */
+@event("toggle")
+class NotificationListGroupItem extends NotificationListItemBase {
+/**
+ * Defines if the group is collapsed or expanded.
+ * @type {boolean}
+ * @defaultvalue false
+ * @name sap.ui.webc.fiori.NotificationListGroupItem.prototype.collapsed
+ * @public
+ */
+@property({ type: Boolean })
+collapsed!: boolean;
+
+/**
+ * Defines if the items <code>counter</code> would be displayed.
+ * @type {boolean}
+ * @defaultvalue false
+ * @name sap.ui.webc.fiori.NotificationListGroupItem.prototype.showCounter
+ * @public
+ */
+@property({ type: Boolean })
+showCounter!: boolean;
+
+/**
+ * Defines the items of the <code>ui5-li-notification-group</code>,
+ * usually <code>ui5-li-notification</code> items.
+ *
+ * @type {sap.ui.webc.fiori.INotificationListItem[]}
+ * @slot items
+ * @name sap.ui.webc.fiori.NotificationListGroupItem.prototype.default
+ * @public
+ */
+@slot({ type: HTMLElement, "default": true })
+items!: Array<HTMLElement>
+
+static get styles() {
+	return NotificationListGroupItemCss;
+}
+
+static get template() {
+	return NotificationListGroupItemTemplate;
+}
+
+onBeforeRendering() {
+	if (this.busy) {
+		this.clearChildBusyIndicator();
+	}
+}
+
+/**
+ * Clears child items busy state to show a single busy over the entire group,
+ * instead of multiple BusyIndicator instances
+ */
+clearChildBusyIndicator() {
+	this.items.forEach(item => {
+	  (item as unknown as { busy: boolean }).busy = false;
```

Comment:
This `(item as unknown as { busy: boolean }).busy` shows that the items slot should be of type NotificationListItem